vllm/engine/arg_utils.py

```diff
@@ -434,6 +438,29 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                             'MoE layers will be sharded according to the '
                             'product of the tensor-parallel-size and '
                             'data-parallel-size.')
+        parser.add_argument('--data-parallel-size-local',
+                            '-dpl',
+                            type=int,
+                            default=EngineArgs.data_parallel_size_local,
+                            help='Number of data parallel replicas to run on '
+                            'this node.')
+        parser.add_argument('--data-parallel-start-rank',
+                            '-dpr',
+                            type=int,
+                            default=EngineArgs.data_parallel_start_rank,
+                            help='Starting data parallel rank for secondary '
+                            'nodes.')
+        parser.add_argument('--data-parallel-address',
```

Comment:
we have `data_parallel_master_ip` and `data_parallel_master_port`. can we use those two field and rename the cli arg? i feel `data_parallel_master_ip` and `data_parallel_master_port` are easier to understand compared to `--data-parallel-address` and `--data-parallel-rpc-port`.

in addition, I reserved 10 ports starting from `data_parallel_master_port`, so you can use, say `data_parallel_master_port + 2`, as `--data-parallel-rpc-port`, which will make the user-interface easier to use.

see https://github.com/vllm-project/vllm/blob/2386803f2a2e3df1f29ea05212eaf68590b85805/vllm/utils.py#L581

Reply:
I could rename `data-parallel-address` to `data-parallel-ip-address`? or `data-parallel-head-ip`? I was also trying to avoid using the "master" term :)

Re the port, the specified port in this case is actually the zmq socket port, the torch process group is assigned an available port arbitrarily (which is propagated to the other nodes).